ignite/services/plugin/internal/execute_test.go

```diff
@@ -0,0 +1,46 @@
+package plugininternal
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ignite/cli/v28/ignite/services/plugin"
+	"github.com/stretchr/testify/require"
```

Comment:
Imports sorting:

```suggestion
	"github.com/stretchr/testify/require"

	"github.com/ignite/cli/v28/ignite/services/plugin"
```

Reply:
re-added ignite/cli in my `goimports -local` flag list :)